File: projects/tox4j/src/test/java/im/tox/gui/ToxGuiClient.java
package im.tox.gui;

import javax.swing.*;

public final class ToxGuiClient {

  /**
   * Run a Tox GUI client with Nimbus L&amp;F.
   */
  public static void main(String[] args) throws Exception {
    for (UIManager.LookAndFeelInfo info : UIManager.getInstalledLookAndFeels()) {
      if ("Nimbus".equals(info.getName())) {
        UIManager.setLookAndFeel(info.getClassName());
        break;
      }
    }

    MainView dialog = new MainView();
    dialog.pack();
    dialog.setVisible(true);
  }

}

File: projects/tox4j/src/test/java/im/tox/gui/ToxGuiClient.scala
package im.tox.gui

import javax.swing._

/**
 * Run a Tox GUI client with Nimbus L&amp;F.
 */
object ToxGuiClient extends App {

  for {
    info <- UIManager.getInstalledLookAndFeels
    if info.getName == "Nimbus"
  } {
    UIManager.setLookAndFeel(info.getClassName)
  }

  val dialog = new MainView
  dialog.pack()
  dialog.setVisible(true)

}
